tests/ui/conftest.py

```diff
@@ -95,29 +297,29 @@ def user(create_superuser, my_base_url, fxa_account, jwt_token):
 @pytest.fixture(scope='function')
 def live_server(request, transactional_db):
```

Comment:
Can we file an issue to investigate this? I suspect if we were able to use the session scoped `live_server` fixture then many of our issues with running tests is parallel would go away.